ome_zarr/writer.py

```diff
@@ -88,13 +89,15 @@ def _validate_well_images(images: List, fmt: Format = CurrentFormat()) -> None:
                 raise ValueError(f"{image} path must be of string type")
             if "acquisition" in image and not isinstance(image["acquisition"], int):
                 raise ValueError(f"{image} acquisition must be of int type")
+            validated_images.append(image)
```

Comment:
Not really related to this PR, but this PR did make me notice that here we throw on the first error, rather than finding all validation errors and listing them.

Reply:
Agreed, this is a fail-fast approach. Worth capturing as an RFE?

Reply:
https://github.com/ome/ome-zarr-py/issues/158